scrapers/StateOfTexas.mjs

```diff
@@ -1,32 +1,41 @@
-import fetch from 'node-fetch';
-import moment from 'moment';
+import fetch from "node-fetch";
+import moment from "moment";
 
 export default ({ database, DataScraper }) => {
-	return DataScraper(database, 'StateOfTexas', async () => {
+	return DataScraper(database, "StateOfTexas", async () => {
 		const data = [];
-		const baseUrl = 'https://api.tnris.org/api/v1';
-		const id = '2679b514-bb7b-409f-97f3-ee3879f34448';
-		const collections = `collections/${id}`;
-		const resources = `resources?collection_id=${id}`;
-		const collectionsRequest = await fetch(`${baseUrl}/${collections}`);
-		const collection = await collectionsRequest.json();
-		const resourcesRequest = await fetch(`${baseUrl}/${resources}`);
-		const resource = await resourcesRequest.json();
+		const baseUrl = "https://api.tnris.org/api/v1";
+		//fetching the collection_id
+		const idData = await fetch(
+			"https://api.tnris.org/api/v1/collections_catalog/"
+		).then((response) => response.json());
```

Comment:
Please be mindful about the consistency of variable names, `response` is used here, but `res` is used underneath.